app/main.py

```diff
@@ -15,3 +15,4 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :return: count occurrences of letter in phrase
     """
     # write your code here
```

Comment:
You should remove this line when implementing solution :)

Reply:
okay) I've fixed

Reply:
sorry, missclick